src/content/docs/pt-br/community-resources/content.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Conteúdo Educacional da Comunidade
+description: Descubra tutoriais, guias, artigos e vídeos produzidos pela comunidade para ajudar você a aprender e contruir com Astro!
+i18nReady: true
+---
+
+import Badge from '~/components/Badge.astro';
+
+Existe muito mais conteúdo lá fora que pode ajudar você a aprender e construir com Astro! Aqui estão alguns conteúdos educacionais construidos e mantidos pela comunidade do Astro.
+
+Todos os links são externos, e podem ser baseados em versões anteriores do Astro. Confira a data em qualquer conteúdo da comunidade, e adapte de acordo com seu projeto .
+
+Para mais conteúdo educacional do Astro, confira [o Blog do Astro](https://astro.build/blog/) ou assine [a newsletter oficial do Astro](https://astro.build/newsletter/signup) onde nós trazemos o melhor conteúdo da comunidade de cada mês.
+
+:::note[Nota dos tradutores]
+Apesar das descrições abaixo estarem traduzidas, o conteúdo dos links é em Inglês.
+:::
+
+## Cursos e Tutoriais
+
+Cursos e tutoriais são focados em ensinar você novos conceitos, geralmente com exercícios ou projetos de exemplo para construir. São ótimas maneiras de aprender novos conceitos e fornecer as ferramentas necessárias para trabalhar com Astro em seu projeto.
+
+Confira os seguintes cursos e tutoriais para aprender mais sobre Astro.
+
+### Tutoriais Introdutórios
+  - [Curso intensivo sobre o framework web Astro](https://www.youtube.com/watch?v=e-hTm5VmofI) por freeCodeCamp
+  - [Curso intensivo sobre Astro em 20 minutos](https://www.youtube.com/watch?v=zrPVTf761OI) por Chris Pennington (curso completo pago: [Aprenda Astro](https://learnastro.dev/))
+  - [Curso intensivo sobre Astro 3.0](https://www.youtube.com/watch?v=qBOz6TpYAOg) por James Q Quick (curso completo pago: [Construa Sites modernos com Astro](https://astrocourse.dev/))
+  - [Curso intenssivo sobre Astro em 60 minutos](https://www.youtube.com/watch?v=NniT0vKyn-E) por @developedbyed
+  - [Entendendo Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) por Ohans Emmanuel
+  - [O Vale do Código - Astro](https://thevalleyofcode.com/astro) por Flavio Copes
+
+  ### Vídeo Tutoriais
```

Comment:
This is very subjective, but I don't see the term "Vídeo Tutoriais"  being used so often, in situations like this the translation goes for "em Vídeo" instead, which for me sounds a bit easier to read, what do you think?
```suggestion
### Tutoriais em Vídeo
```

Reply:
Makes total sense! Thanks for catching this 😇